.env.local

```
@@ -1,3 +1,4 @@
 POSTGRES_USER=ageraldo
 POSTGRES_PASSWORD=dev123
 POSTGRES_DB=gobarber
+SALT=d74c2ec2961bf5df1e8411cdfa2b8f0f
```

Ainda não estou usando mas logo, mas será importante quando tiver no ambiente de produção só lembra de comentar que a chave será trocada.
```suggestion
#The key is another one in production
SALT=d74c2ec2961bf5df1e8411cdfa2b8f0f 
```